pyvo/dal/scs.py

```diff
@@ -87,7 +87,7 @@ class SCSService(DALService):
     a representation of a Cone Search service
     """
 
-    def __init__(self, baseurl, session=None):
+    def __init__(self, baseurl, capability_description=None, session=None):
```

Comment:
Unfortunately, you can't do this (not if this goes in before #507) as it changes the API in a backward incompatible way.
(part of the motivation for #507 is to make it easier to add new parameters to signatures, and insert them wherever it makes most sense, without that PR, new parameters always have to go to the end)